### PR TITLE
Prepare 'envelopes' queue configuration for introduction of another queue

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-452

### Change description ###

Prepare 'envelopes' queue configuration for introduction of another queue.
Also, remove some unused configuration bits

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -59,8 +59,8 @@ withPipeline(type, product, component) {
       def serviceBusConnectionStr = kubectl.getSecret(serviceBusSecret, namespace, "{.data.connectionString}")
       //Regex: Endpoint=sb:\/\/([-a-zA-Z0-9]+)\.servicebus\.windows\.net.*
       def serviceBusNamespace = serviceBusConnectionStr.substring(14).split("\\.")[0]
-      env.QUEUE_READ_CONN_STRING = "${serviceBusConnectionStr};EntityPath=envelopes"
-      env.QUEUE_WRITE_CONN_STRING = "${serviceBusConnectionStr};EntityPath=envelopes"
+
+      env.ENVELOPES_QUEUE_WRITE_CONN_STRING = "${serviceBusConnectionStr};EntityPath=envelopes"
```
Comment (rkondratowicz) [inline Jenkinsfile_CNP]: should it not be the 'read' conn string?

File: README.md [hunk 2]
```diff
@@ -14,12 +14,13 @@ IDAM_USERS_BULKSCAN_PASSWORD = Password12
...
-    connection-string: XXXXX
-    read-interval: YYYY
+    envelopes:
+      connection-string: XXXXX
+      queue-name: YYYY
   ```
   - where: 
     - XXXX is the connection string from azure to the queue you intend to use.
-    - YYYY is optional but recommended to reduce this to 500 ms for speed of development.
+    - YYYY is the name of the queue
```
Comment (rkondratowicz) [inline README.md]: what's `envelopes` then?

File: Jenkinsfile_CNP (hunk 1 above)
Comment (rkondratowicz) [reply Jenkinsfile_CNP]: or is that for tests?

Comment (lgonczar) [reply Jenkinsfile_CNP]: It should be `write` - it's for tests, not the app

File: README.md (hunk 2 above)
Comment (lgonczar) [reply README.md]: Envelopes is the queue named `envelopes`. Alternatively, I can rename the prefixed in this PR to sth like `incoming-envelopes`. The new one will be `processed-envelopes`.

**Comment by lgonczar** [comment]
> so we are replacing read connection with write for.. reading part? erm.. this pr looks like a clash of two similar tasks

I confused everyone. There are actually two changes in this PR, in separate commits. One change adds ENVELOPES_ prefix (more or less) and the other one removes unused configuration.

**Pull request merged**